src/common/utils.js

```diff
@@ -35,12 +35,13 @@ function toRippledAmount(amount: Amount): RippledAmount {
   };
 }
 
-const FINDSNAKE = /([a-zA-Z]_[a-zA-Z])/g;
 function convertKeysFromSnakeCaseToCamelCase(obj: any): any {
   if (typeof obj === 'object') {
     let newKey;
     return _.reduce(obj, (result, value, key) => {
       newKey = key;
+      // taking this out of function leads to error in PhantomJS
```

Comment:
Yes, they should. But in PhantomJS it has problems. I've moved it back so you could see errors by yourself.
And I don't think that we testing it wrong - I think it is problem inside PhantomJS. As they documentation states, PhantomJS in not just headless chrome, it is custom build with some version of V8, that, maybe, has some bugs, or linked with bugs.
We will be more sure when I will add Sauce Labs testing, so we can see test results in real browsers.